### ISSUE TITLE
Tag groups

### ISSUE THREAD

**Issue by FR6** [rtconner/laravel-tagging#95]
Can we create "tag groups"?

For example, i would like to be able to create a sets of tags "colors" and "sizes" associated to a model "blog post".

Currently all tags will have the same "taggable_type" (blog post) so they all will be considered into the same group. 

Seems related to issue #83 .

**Comment by thaonguyen90** [comment]
waiting a new version :+1:

**Comment by arippberger** [comment]
+1

**Comment by elephantux** [comment]
+1

**Comment by jbardnz** [comment]
Hey guys,

Tag Groups is a feature I need as well so I have created it in a [fork here.](https://github.com/jbardnz/laravel-tagging)

So far I have just added basic functionality so tags can belong to groups, and you have a InGroup scope.
`$tags = Tag::inGroup('group_name')->get();`

I am looking for other features I should add?

I will make a pull request back to this repo once it is complete but I am not sure if this package is still active?
